example.py

```diff
@@ -32,13 +32,13 @@ def demo():
         return
 
     # Fetch datas
-    lvbx.system.get_deviceinfo()
+    lvbx.deviceinfo.get_deviceinfo()
     lvbx.connection.get_data_MIBS()
     lvbx.connection.get_dsl0_MIBS()
     lvbx.system.get_guest()
 
-    lip = (lvbx.connection.get_lan_luckyAddrAddress())["status"]
-    wip = (lvbx.connection.get_data_luckyAddrAddress())["status"]
+    lip = (lvbx.connection.get_lan_luckyAddrAddress())["status"]  # type: ignore[index]
+    wip = (lvbx.connection.get_data_luckyAddrAddress())["status"]  # type: ignore[index]
```

Comment:
The result could be `None`.
Maybe it should be guarded, but it's an example file so it should be fine.